### PR TITLE
Add assignee logic

### PR DESCRIPTION
**Pull request by Notalot**
- Add assignee logic
- Add client/server info to summary

### CODE REVIEW

**Review by BigBadBLOO — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const filterUnresolvedDeps = async (config, dependencies, devDependencies) => {
     email: config.email,
   });
 
-  const jiraTasks = await jira.searchDepcheckIssues({comment: config.comment, summary: `${github.context.repo.repo} ${SUMMARY}`});
+  const jiraTasks = await jira.searchDepcheckIssues({comment: config.comment, summary: `${github.context.repo.repo} - ${SUMMARY} ${config.comment}`});
```
Comment (fanmilan) [inline index.js]: I don't remember why but jira search doesnt' work well with `-`

image
image

Comment (fanmilan) [reply index.js]: but without spaces it works) weird...
image

Comment (fanmilan) [reply index.js]: https://jira.atlassian.com/browse/JRACLOUD-72895
there is the same problem
image
so you can you use it with double slash

Comment (Notalot) [reply index.js]: done, thank you!

**Review by fanmilan — APPROVED** [review]

File: index.js
Comment (fanmilan) [inline index.js:1]: @Notalot Wait? Don't we need to rebuild index.js?

Comment (Notalot) [reply index.js:1]: oops, how to do that?

Comment (Notalot) [reply index.js:1]: done, thnx

**Review by fanmilan — APPROVED** [review]

**Pull request merged**